non-critical-infra/modules/mailserver/mailing-lists.nix

```diff
@@ -0,0 +1,39 @@
+# This module provides the mailing list definitions for `@nixos.org`.
+#
+# Simply change the `lists` attribute set below to create new mailing lists or
+# edit membership of existing lists.
+#
+# If you wish to hide your email address, you can encrypt it with SOPS. Just
+# run `nix run .#encrypt-email-address` in the `non-critical-infra/` folder and
+# follow the instructions.
+
+{ config, lib, ... }:
+
+let
+  # Mailing lists go here.
+  # TODO: replace with the real `nixos.org` mailing lists.
+  lists = {
+    "jfly@playground.nixos.org" = [
+      "jfly@playground.jflei.com"
+      config.sops.placeholder.jfly-email
+      "jeremyfleischman+subscriber@gmail.com"
+    ];
+  };
+in
+
+{
+  # Encrypted email addresses go here.
+  sops.secrets.jfly-email = {
+    format = "binary";
+    sopsFile = ../../secrets/jfly-email.umbriel;
```

Comment:
Maybe we want to put this in some subdirectory and import all files in it using `builtins.readDir`. Than adding new emails will be easier.

Reply:
I totally agree. I've reworked this. It's not quite what you suggest: instead now the list of mailing list members contains either strings or files. I then extract the files from the member lists to declare the sops secrets.